config/helm/chart/default/templates/Common/operator/deployment-operator.yaml

```diff
@@ -94,6 +97,9 @@ spec:
                     operator: In
                     values:
                       - linux
+      volumes:
+        - emptyDir: { }
+          name: tmp-dir
```

Comment:
nitpick: call it something more descriptive, we will forget what this tmp-dir does in 2 weeks :D
maybe `tmp-cert-dir`